package.json

```diff
@@ -7,16 +7,17 @@
     "oc": "./src/oc-cli.js"
   },
   "scripts": {
-    "build": "node tasks/build.js && npm run test",
-    "changelog": "node tasks/changelog.js",
+    "build": "node tasks/build.js",
     "git-stage": "node tasks/git-stage.js",
-    "major": "node tasks/version.js --type=\"major\"",
-    "minor": "node tasks/version.js --type=\"minor\"",
-    "patch": "node tasks/version.js --type=\"patch\"",
+    "major": "npm run version-major && npm run test-silent && npm run git-stage",
+    "minor": "npm run version-minor && npm run test-silent && npm run git-stage",
+    "patch": "npm run version-patch && npm run test-silent && npm run git-stage",
```

Comment:
you can also use the native `npm version <update_type>` to update version on npm btw

Reply:
This is a bit different as it updates the component's version too.